include/mbedtls/cipher.h

```diff
@@ -1080,7 +1076,7 @@ int mbedtls_cipher_crypt(mbedtls_cipher_context_t *ctx,
                          const unsigned char *input, size_t ilen,
                          unsigned char *output, size_t *olen);
 
-#if defined(MBEDTLS_CIPHER_MODE_AEAD) || defined(MBEDTLS_NIST_KW_C)
+#if defined(MBEDTLS_CIPHER_HAVE_SOME_AEAD) || defined(MBEDTLS_NIST_KW_C)
```

Comment:
Is this macro possibly defined at this point? Or is it not visible because it's define in `cipher_wrap.c` now?

Actually, taking a second look: `defined(MBEDTLS_GCM_C) || defined(MBEDTLS_CCM_C) || defined(MBEDTLS_CHACHAPOLY_C)` is the right macro here: this function won't work with drivers, so we really want the guard to ask for the built-in implementations.